### ISSUE TITLE
FF keyboard nav delay(?)

### ISSUE THREAD

**Issue by phet-steele** [phetsims/balloons-and-static-electricity#139]
In Win 10 FF, once a balloon has charges its WASD control breaks. It's hard to explain, but it *feels* like the "window" for another keyboard input closes for a short time once a key is pressed.
@jessegreenberg I don't remember this happening before, my guess is the recent FF update to 45.0.2 caused this. For phetsims/tasks/issues/566.

A video that hopefully makes it clear:
https://drive.google.com/file/d/0B3HJopSo_QqLTVJSOTViNmwtSFU/view?usp=sharing

**Comment by terracoda** [comment]
@jessegreenberg I witnessed this behaviour (severely) in one interview, but strangely not in all the interviews that used the same computer. I didn't know how to describe it. I called it the "pendulum effect."

**Comment by terracoda** [comment]
Sorry, that last comment should have been addressed to both @jessegreenberg  and @phet-steele.

**Comment by terracoda** [comment]
@phet-steele and @jessegreenberg I also witnessed delays  and catch-ups in an interview with a Surface Pro 3 Tablet.

**Comment by jessegreenberg** [comment]
After trying for about 5 minutes, I cannot reproduce on Windows 10 FF 45.0.2.  I have observed this once before on another platform.

**Comment by jessegreenberg** [comment]
I just realized that if you release the balloon or pull up the keyboard help while holding down arrow keys, the balloon will continue to move in the direction of the arrow key once you pick it up again, and will continue to move in that direction until you press that arrow key once more. I think this is a similar issue, but unrelated to the original post.

**Comment by jessegreenberg** [comment]
When the balloon is dropped or receives the blur event, the keystate object should be reset.

**Comment by jessegreenberg** [comment]
Closing, we have had an entirely new implementation since this was opened and this no longer has relevance.
